AdyenActions/Components/Await/AwaitComponent.swift

```diff
@@ -26,7 +26,7 @@ public final class AwaitComponent: ActionComponent, Cancellable {
     public struct Configuration {
         
         /// The component UI style.
-        public var style: AwaitComponentStyle = .init()
+        public var style: AwaitComponentStyle
```

Comment:
nice! can we also change this in 
- `DocumentComponent.swift` line 43
- `QRCodeComponent.swift` line 53
- `VoucherComponent.swift` line 53
- `AwaitComponent.swift` line 47
- `ThreeDS2Component.swift` line 28

Reply:
done